src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -640,7 +641,7 @@ def _get_custom_or_builtin_policy(cmd, client, name, subscription=None, manageme
         if cmd.supported_api_version(min_api='2018-03-01'):
             if not management_group:
                 management_group, name = _parse_management_group_reference(name)
```

Comment:
In `_parse_management_group_reference()`, when `_is_management_group_scope() == true` and `len(parts) < 9`, the name will be set to null affecting the call to `policy_operations.get(name)` below.

Reply:
good catch, I'm trying to overload too much logic in this one parse method